pub/health_check.php

```diff
@@ -43,6 +43,10 @@
 $cacheConfigs = $deploymentConfig->get(ConfigOptionsListConstants::KEY_CACHE_FRONTEND);
 if ($cacheConfigs) {
     foreach ($cacheConfigs as $cacheConfig) {
+        unset($cacheConfig['id_prefix']);
+        if (empty($cacheConfig)) {
+            break;
```

Comment:
Could you add more details on this break? Why we allow empty configuration?

Reply:
@orlangur as in later version in env.php we have no page_cache with "id_prefix" , as now in env.php we have a static "id_prefix" wich always present in config, and we should check if config contain only id_prefix then we should allow that all is OK, but if we unset "id_prefix" and have not empty configuration it will set 500 errors,

Reply:
In one word we allow that configuration contains id prefix.

Reply:
Why need unset them? Can we just check that "there is only key - id_prefix - then allow"?

Reply:
@orlangur ok this is my 1 solution 
```
 if (count($cacheConfig) === 1 && isset($cacheConfig['id_prefix'])) {
            break;
        }
```
but then i thinking that this is not proper :)

Reply:
is this solution is better ?